apps/mobile/mobile-app/components/StyledText.tsx

```
@@ -2,4 +1,249 @@
+        <View style={styles.textContainer}>
+          <Text style={styles.title}>{item.title}</Text>
+          <Text style={styles.description}>{item.description}</Text>
+
+          {/* Progress Section */}
+          <View style={styles.progressContainer}>
+            <View style={styles.progressTextContainer}>
+              <Text style={styles.progressAmount}>{item.progress.amountRaised}</Text>
+              <Text style={styles.progressPercentage}>{item.progress.percentage} of {item.progress.goal}</Text>
+            </View>
+            <View style={styles.progressBarBackground}>
+              <View style={[styles.progressBarFill, { width: `${parseFloat(item.progress.percentage)}%` }]} />
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)